app/tests/test_models.py

```diff
@@ -1,7 +1,39 @@
+from flask import Flask
+from flask.ext.testing import TestCase
 from nose.tools import eq_
 
 from .. import models
 
+db = models.db
+
+class DbTestCase(TestCase):
+    BASE_APP_CONFIG = dict(
+        # TODO: We should use a test postgres db instead, since it's
+        # closer to our production deployment.
```

Comment:
I would do it as part of the test suite setup.

However, I would not use a sqlite database when testing.  We have easy access to postgres thanks to our using Docker, and there are significant differences in behavior between sqlite and postgres.